src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -48,15 +48,16 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     /// <typeparam name="TKey">The key type to be used by the cache.</typeparam>
     /// <typeparam name="TValue">The value type to be used by the cache</typeparam>
-    internal class EventBasedLRUCache<TKey, TValue> : IDisposable
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1001:TypesThatOwnDisposableFieldsShouldBeDisposable")]
```

Comment:
Please add the Justification in the SuppressMessage.

Reply:
We usually put suppression messages in the GlobalSuppressions.cs file. Is there a reason why it's in the code?